src/eva/plot_tools/figure.py

```diff
@@ -164,7 +164,7 @@ def _map_scatter(self, plotobj, ax):
 
             cs = ax.scatter(plotobj.longitude, plotobj.latitude,
                             s=plotobj.markersize, **inputs,
-                            transform=self.projection.projection)
+                            transform=ccrs.PlateCarree())
```

Comment:
Do we want to start hardwiring the projection?

Reply:
Yeah... we don't want to hard code these sort of things...

Reply:
Is there any way to pass ccrs.PlateCarree() through self.projection.projection? I tried but failed to figure out.

Reply:
See the MapProjection class...: https://github.com/JCSDA-internal/eva/blob/18f1225feba26fb40949753d5177f1c2080e18c6/src/eva/plot_tools/maps.py#L292